server/src/main/java/org/elasticsearch/http/netty4/Netty4HttpServerTransport.java

```diff
@@ -235,7 +235,6 @@ public Netty4HttpServerTransport(Settings settings,
                                      PipelineRegistry pipelineRegistry,
                                      NettyBootstrap nettyBootstrap,
                                      NodeClient nodeClient) {
-        Netty4Utils.setAvailableProcessors(EsExecutors.PROCESSORS_SETTING.get(settings));
```

Comment:
Why is this not needed anymore?
If valid, method could probably removed as it has no usage anymore

Reply:
I had moved it to NettyBootstrap but forgot to save the file :man_facepalming: 

Pushed a fixup